src/Admin/AbstractAdmin.php

```diff
@@ -2063,7 +2063,22 @@ public function setCurrentChild($currentChild)
         $this->currentChild = $currentChild;
     }
 
+    /**
+     * NEXT_MAJOR: Remove this method
+     *
+     * @deprecated since sonata-project/admin-bundle 3.x, to be removed in 4.0
+     */
     public function getCurrentChild()
+    {
+        @trigger_error(
+            'The '.__METHOD__.' method is deprecated since version 3.x and will be removed in 4.0. Use isCurrentChild instead.',
+            E_USER_DEPRECATED
+        );
```

Comment:
```suggestion
        @trigger_error(sprintf(
            'The %s() method is deprecated since sonata-project/admin-bundle 3.x and will be removed in 4.0. Use %s::isCurrentChild() instead.',
            __METHOD__,
            __CLASS__
        ), E_USER_DEPRECATED);
```

Reply:
Done @phansys